HealthcareSystemFrontends/GraphicalEditor/DTO/EmergencyExaminationsDTO.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GraphicalEditor.DTO
+{
+    public class EmergencyExaminationsDTO
+    {
+        public List<EmergencyExaminationDTO> EmergencyExaminations { get; set; }
+        public bool Shifted { get; set; }
```

Comment:
We can delete this class now since it is no longer in use.

Reply:
I deleted it in this commit.